inc/db.php

```diff
@@ -9,27 +10,33 @@ public static function init() {
 			Config::$dbPassword
 		);
 		static::$conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+		static::$isMysql = static::$conn->getAttribute(PDO::ATTR_DRIVER_NAME) == 'mysql';
 		static::createTables();
 	}
 
 	private static function createTables() {
 		static::$conn->exec('
 			CREATE TABLE IF NOT EXISTS packages (
-				PackageId TEXT PRIMARY KEY,
-				Title TEXT,
+				PackageId ' . (static::$isMysql ? 'VARCHAR (256)' : 'TEXT PRIMARY KEY') . ',
```

Comment:
SQLite also supports `VARCHAR` and treats it the same as `TEXT` - You can change this to `VARCHAR(256) for both.